core/src/main/java/com/google/errorprone/internal/NonDelegatingClassLoader.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.errorprone.internal;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.net.URLClassLoader;
+import java.util.Set;
+
+/**
+ * A non-delegating {@link java.net.URLClassLoader} that searches it's own resource path _before_
```

Comment:
Fixed.

Heh, github markdown recognized `_before_` and `<i>before<i>` as _before_.